plugins/vlc-video/vlc-video-plugin.c

```diff
@@ -179,9 +179,9 @@ static bool load_libvlc_module(void)
 	RegCloseKey(key);
 #else
 
+/* According to otoolo -L, this is what libvlc.dylib wants. */
 #ifdef __APPLE__
 #define LIBVLC_DIR "/Applications/VLC.app/Contents/MacOS/"
-/* According to otoolo -L, this is what libvlc.dylib wants. */
 #define LIBVLC_CORE_FILE LIBVLC_DIR "lib/libvlccore.dylib"
```

Comment:
Weird, something should be done to avoid this format.

Reply:
That's a manual fix, because clang-format will otherwise indent the comment. Overall the new version seems to be very opinionated about preprocessor blocks (one more reason to remove as much of it as possible).

Reply:
It might be worth clarifying the comment then to specify "what libvlc.dylib wants for LIBVLC_CORE_FILE" or some such.

Reply:
I have no idea what that means, I just moved the position of the comment to avoid unnecessary indentation.

Reply:
There's not much context given for the original addition of the comment.
https://github.com/obsproject/obs-studio/commit/ec366cfa3d178760c26ac303dffe98ed4e0abd9f

While I wouldn't mind extra clarification or a TODO to clarify later, I'm not going to block on this. Git blame will show the change and the commit, which will lead to this PR.